Twig/Extension/FormatterMediaExtension.php

```diff
@@ -109,4 +109,14 @@ public function path($media, $format)
     {
         return $this->getTwigExtension()->path($media, $format);
     }
+
+    /**
+     * NEXT_MAJOR: Remove this method when bumping requirements to twig 2.
+     *
+     * {@inheritdoc}
+     */
+    public function getName()
```

Comment:
This PR allows to use Twig 2. It does not force to use Twig 2. So yeah, we still need these methods until we require Twig 2 ourselves.